src/main/flight/pid.c

```diff
@@ -86,6 +88,12 @@ PG_RESET_TEMPLATE(pidConfig_t, pidConfig,
 
 PG_REGISTER_ARRAY_WITH_RESET_FN(pidProfile_t, MAX_PROFILE_COUNT, pidProfiles, PG_PID_PROFILE, 2);
 
+#ifdef USE_TXPID
+// Scale txPID value around mid-point assuming RC input range of 1000-2000
+#define TX_PID_VAL(PID, TERM) pidProfile->centerVal[PID][TERM] + \
```

Comment:
Function will be cleaner

Reply:
Done. More verbose, but as you say, clearer.